pkgs/development/compilers/zulu/24.nix

```diff
@@ -5,52 +5,49 @@
 }@args:
 
 let
-  # For 24 JDK FX is newer than regular JDK
-  zuluVersion = if enableJavaFX then "24.30.13" else "24.30.11";
+  # For 24 JDK FX can be different version than regular JDK
+  zuluVersion = if enableJavaFX then "24.32.13" else "24.32.13";
```

Comment:
Should these be the same?

Reply:
Yes, they should be the same. I thought about removing that line entirely, but as they are sometimes different, I thought I would leave it there. Note that I changed the comment from "is newer" to "can be different" to reflect that they are currently the same and that I had observed that FX can also be older.